### PR TITLE
Grid styles

### PR DESCRIPTION
**Pull request by gestchild**
just added some more styles; allows for the different column numbers at different breakpoints, should we need them

### CODE REVIEW

File: client/scss/base/_container.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.container {
+  margin: 0 auto;
+  width: map-get($container-width, 'small');
+  padding: 0  map-get($container-padding, 'small') 0;
```
Comment (jamesgorrie) [inline client/scss/base/_container.scss]: Do we need the last `0` here?

File: server/views/layout/default.njk [hunk 2]
```diff
@@ -9,6 +9,10 @@
 </head>
 
 <body>
-    {% block content %}{% endblock %}
+  <div class="container">
+    {% block body %}
+      default content
```
Comment (jamesgorrie) [inline server/views/layout/default.njk]: Could we remove this as we'd never want this to show.

**Comment by jamesgorrie** [comment]
Couple of comments, but looks good.

File: client/scss/base/_container.scss (hunk 1 above)
Comment (gestchild) [inline client/scss/base/_container.scss]: Nope, I'll take it out

File: client/scss/grids/_grid_position.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  align-self: flex-start;
+}
+
+.grid__cell--bottom {
+  align-self: flex-end;
+}
+
+.grid__cell--center {
+  align-self: center;
+}
```
Comment (davidpmccormick) [inline client/scss/grids/_grid_position.scss]: Can you add a rule to get your editor to ensure there's a final newline at the end of files?

File: client/scss/utilities/_variables.scss [hunk 4]
```diff
@@ -8,16 +8,21 @@ $font-large: 2.6rem;
...
 
 $container-padding: (
-  'large': 60px,
+  'small': 30px,
   'medium': 42px,
-  'small': 30px
+  'large': 60px
 );
 
 $gutter-width: (
```
Comment (davidpmccormick) [inline client/scss/utilities/_variables.scss]: Should we get rid of this or is it needed somewhere else?

File: server/views/patterns/typography.njk [hunk 5]
```diff
@@ -1,27 +1,379 @@
...
-        <li>item one</li>
-        <li>item two</li>
-        <li>item three</li>
-        <li>item four</li>
-    </ul>
+{% extends "views/layout/patterns.njk" %} {% block body %}
+<article>
+  <header>
+    <h1>Typography Style Guide</h1>
+    <p class="lede">Basic markup and typographic styles</p>
```
Comment (davidpmccormick) [inline server/views/patterns/typography.njk]: Should that be `lead`?

File: server/views/patterns/typography.njk [hunk 6]
```diff
@@ -1,27 +1,379 @@
...
     </ol>
-</div>
+  </div>
+
+  <h3>Unordered list</h3>
+  <p>The <code>ul</code> element denotes an unordered list (ie. a list of loose items that don&#8217;t require numbering, or a bulleted list). Again, each item requires a surrounding <code>&lt;li&gt;</code> and <code>&lt;/li&gt;</code> tag, to denote individual items. Here is an example list showing the constituent parts of the British Isles:</p>
+  <div class="example">
+    <ul>
+      <li>
+        United Kingdom of Great Britain and Northern Ireland:
```
Comment (davidpmccormick) [inline server/views/patterns/typography.njk]: Might need updating soon 😨

File: client/scss/grids/_grid_position.scss (hunk 3 above)
Comment (gestchild) [inline client/scss/grids/_grid_position.scss]: Done

**Pull request merged**